devices/ikea.js

```diff
@@ -152,55 +150,42 @@ module.exports = [
         model: 'LED1924G9',
         vendor: 'IKEA',
         description: 'TRADFRI bulb E27 CWS 806 lumen, dimmable, color, opal white',
-        extend: extend.light_onoff_brightness_color(),
-        ota: ota.tradfri,
-        meta: {supportsHueAndSaturation: false},
-        onEvent: bulbOnEvent,
+        extend: tradfriExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
>  I know it does not work for the "old" RGB bulbs, but it does work for the new GU10, E14 and E27.

I don't understand the reason why color temperature is added here.

Reply:
Its 110% supported in CWS3 as being sad many times.
![IKEA-CWS3](https://user-images.githubusercontent.com/49618193/118400027-365b4980-b660-11eb-8e74-aa2c4f9ccd7a.PNG)
And the color loop is working 2 !

Reply:
Ah sorry, was confused, I have a `TRADFRI bulb E27 CWS opal 600lm` and this device doesn't support it.